src/Mod/Image/App/PreCompiled.h

```diff
@@ -34,7 +34,7 @@
 #endif
 
 #ifdef _PreComp_
-/// here get the warnings of to long specifiers disabled (needed for VC6)
+/// here get the warnings of too long specifiers disabled (needed for VC6)

```

Comment:
This PR is not generally applying grammar fixes, however in a couple of cases when a spelling error was identified on a line, in an effort to decipher the intent of the line, I also tried to address the rest of the line.